Adds INP note, links to optimize long tasks guide. (#9141)
GoogleChrome/web.dev: src/site/content/en/blog/long-tasks-devtools/index.md

Large scripts are often a major cause of Long Tasks so consider [splitting them up](/reduce-javascript-payloads-with-code-splitting). Also keep an eye on third-party scripts; their Long Tasks can delay primary content from getting interactive.

> Break all your work into small chunks (that run in < 50ms) and run these chunks at the right place and time; the right place may even be off the main thread, in a worker. Phil Walton's [Idle Until Urgent](https://philipwalton.com/articles/idle-until-urgent/) is a good read on this topic. Jeremy Wagner also has [an article](/optimize-long-tasks/) with strategies for optimizing and breaking up long tasks in general.

Reviewer: As discussed:

```suggestion
Break all your work into small chunks (that run in < 50ms) and run these chunks at the right place and time; the right place may even be off the main thread, in a worker. Phil Walton's [Idle Until Urgent](https://philipwalton.com/articles/idle-until-urgent/) is a good read on this topic. See also the [optimize long tasks article](/optimize-long-tasks/) for general strategies for managing and breaking up long tasks.
```
